### PR TITLE
Fix cli doc seed example link and command

### PR DESCRIPTION
**Pull request by renansoares**
- Fix broken link
- Fix wrong cli command

### CODE REVIEW

**Comment by jacebenson** [comment]
This command is actually... deprecated..

I think this should also update command to be `yarn rw prisma db seed`

image

File: docs/cliCommands.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ Seed your database with test data.
 yarn redwood db seed
```
Comment (jacebenson) [inline docs/cliCommands.md:298]: `yarn rw prisma db seed`

Comment (renansoares) [reply docs/cliCommands.md]: I also fixed the command. thank you

**Review by peterp — APPROVED** [review]

**Comment by peterp** [comment]
Thanks!

**Review by jacebenson — APPROVED** [review]

**Comment by thedavidprice** [comment]
Thanks @renansoares!

**Pull request merged**